modules/installation-osp-deploying-bare-metal-machines.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+ifeval::["{context}" == "installing-openstack-installer-custom"]
+:osp-ipi:
+endif::[]
+
+[id="installation-osp-deploying-bare-metal-machines_{context}"]
+= Deploying a cluster with bare metal machines
+
+If you want your cluster to use bare-metal machines, modify the
+ifdef::osp-ipi[`install-config.yaml`]
+ifndef::osp-ipi[`inventory.yaml`]
+file. Your cluster can have both control plane and compute machines running on bare metal, or just compute machines.
+[NOTE]
+====
+Be sure that your `install-config.yaml` file reflects whether the {rh-openstack} network that you use for bare metal workers supports floating IP addresses or not.
+
+====
+[NOTE]
+====
+Bare-metal compute machines are not supported on clusters that use Kuryr.
+====
+
+.Prerequisites
+
+* The {rh-openstack} link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/index[Bare Metal service (Ironic)] is enabled and accessible via the {rh-openstack} Compute API.
+
+* Bare metal is available as link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/sect-configure#creating_the_bare_metal_flavor[an {rh-openstack} flavor].
+
+* The {rh-openstack} network supports both VM and bare metal server attachment.
+
+* Your network configuration does not rely on a provider network. Provider networks are not supported.
+
+* To deploy the machines on a pre-existing network, an {rh-openstack} subnet must be provisioned.

```

Comment:
These read differently than the others. Is there some way to parallelize these prerequisites or reshuffle?